packages/paste-website/src/pages/utilities/index.mdx

```diff
@@ -13,19 +13,19 @@ import {Callout, CalloutTitle, CalloutText} from '../../components/callout';
 # Utilities Overview
 
 <Callout>
-  <CalloutTitle as="h4">Don't see a utility or component you need listed here?</CalloutTitle>
+  <CalloutTitle as="h4">Don't see a utility, component, or primitive you need listed here?</CalloutTitle>
   <UnorderedList>
     <ListItem>
       First off, we recommend{' '}
       <Anchor href="https://github.com/twilio-labs/paste/issues">creating an issue on GitHub</Anchor> with your
-      component / utility proposal.
+      component, primitive, or proposal.
     </ListItem>
     <ListItem>
       While not all requests are accepted for the design system, we may be able to suggest suitable alternatives - for
       this, drop us a note on #help-design-system or join our Weekly Office hours.
     </ListItem>
     <ListItem>
-      We've also writtten a guide for{' '}
+      We've also written a guide for{' '}
```

Comment:
Add some text below the heading before this callout which answers the question "What is a primitive"? @SiTaggart  can provide copy

Reply:
This isn't the primitive page though...